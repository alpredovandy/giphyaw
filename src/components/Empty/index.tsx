const Empty = () => (
    <div className="card w-full bg-gradient-to-b from-red-50 to-red-100 shadow-xl">
        <div className="card-body items-center text-red-500">
            <h2 className="card-title">Data not found!</h2>
        </div>
    </div>
);

export default Empty;
