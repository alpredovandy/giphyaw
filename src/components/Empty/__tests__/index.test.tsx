import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import Empty from '@/components/Empty';

describe('Test components: Empty', () => {
    const renderEmpty = () => render(<Empty />);

    it('Snapshot', () => {
        const { container } = renderEmpty();

        expect(container).toMatchSnapshot();
    });
});
