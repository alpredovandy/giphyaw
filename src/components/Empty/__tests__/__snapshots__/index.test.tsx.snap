// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Test components: Empty > Snapshot 1`] = `
<div>
  <div
    class="card w-full bg-gradient-to-b from-red-50 to-red-100 shadow-xl"
  >
    <div
      class="card-body items-center text-red-500"
    >
      <h2
        class="card-title"
      >
        Data not found!
      </h2>
      <p>
        Please search or click tab
      </p>
    </div>
  </div>
</div>
`;
